xbmc/cores/AudioEngine/Sinks/pipewire/PipewireStream.h

```diff
@@ -53,16 +53,9 @@ class CPipewireStream
   static void Process(void* userdata);
   static void Drained(void* userdata);
 
-  const pw_stream_events m_streamEvents = {.version = PW_VERSION_STREAM_EVENTS,
-                                           .destroy = nullptr,
-                                           .state_changed = StateChanged,
-                                           .control_info = nullptr,
-                                           .io_changed = nullptr,
-                                           .param_changed = nullptr,
-                                           .add_buffer = nullptr,
-                                           .remove_buffer = nullptr,
-                                           .process = Process,
-                                           .drained = Drained};
+  pw_stream_events CreateStreamEvents() const;
```

Comment:
Function should be static instead of const.

Reply:
updated